WordPress-VIP-Go/ruleset.xml

```diff
@@ -14,8 +14,8 @@
 		<type>error</type>
 		<message>File system writes only work in /tmp/ and inside the /uploads/ folder on VIP Go. To do filesystem writes you must use the WP_Filesystem class, using functions such as %s() won't work or will return unexpected results. Read more here: https://vip.wordpress.com/documentation/using-wp_filesystem-instead-of-direct-file-access-functions/</message>
 	</rule>
-	<rule ref="WordPressVIPMinimum.Functions.RestrictedFunctions.file_ops_fwrite">
-		<severity>6</severity>
+	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read_file_put_contents">
+		<type>error</type>
```

Comment:
Since this ruleset inherits the `WordPressVIPMinimum` one, where this overriding error is set (along with the other function), then it's not needed here as well.